mcs/class/corlib/Test/Microsoft.Win32/RegistryKeyTest.cs

```diff
@@ -166,7 +166,6 @@ public void OpenSubKey_Name_MaxLength ()
 				Assert.IsNull (ex.InnerException, "#C3");
 				Assert.IsNotNull (ex.Message, "#c4");
 				Assert.IsTrue (ex.Message.IndexOf ("255") != -1, "#C5");
-				Assert.IsNull (ex.ParamName, "#C6");
```

Comment:
We can't put in a not-null assert for this instead because it only applies to the new CoreFX registry code right?

Reply:
@akoeplinger this PR basically imports Registry only for Windows, the rest of the platforms use the old implementation (in CoreFX ParamName is not null)